runner/.gitignore

```diff
@@ -0,0 +1,3 @@
+*backup
+terraform*
```

Comment:
it would be better to keep tfstate on a remote provider - https://www.terraform.io/docs/backends/types/s3.html

Reply:
fixed